src/screens/AccountScreens/ProfileScreen/index.js

```diff
@@ -7,34 +7,65 @@ import { Avartar } from '../../../components/Avatar';
 import { isValidEmail } from '../../../components/utils';
 import { useProfileStore } from '../../../shared/zustand/profile';
 import { useAuthStore } from '../../../shared/zustand/auth';
+import Toast from 'react-native-toast-message';
+import { TOAST_UP_OFFSET } from '../../../components/constants';
+import { getDataObj } from '../../../shared/utils/asyncStorage';
+import { USER_INFO_SESSION_STORAGE_FIELD } from '../../../shared/api/auth/constants';
```

Comment:
`Screens` should only interact with Zustand stores, rather than specific API functions.
The logic of updating cached data should be packed and handled in functions defined in respective stores. This helps us better manage the codebase and reduce coupling.